src/util/cilfacade.ml

```diff
@@ -251,15 +251,27 @@ let ptrdiff_ikind () = get_ikind !ptrdiffType
     instead of printing all errors directly... *)
 
 type typeOfError =
-  | RealImag_NonNumerical (** unexpected non-numerical type for argument to __real__ *) (* TODO: or __imag__? *)
+  | RealImag_NonNumerical (** unexpected non-numerical type for argument to __real__/__imag__ *) (* TODO: CIL's own output forgets __imag__ *)
```

Comment:
`__imag__` as added by ourselves, we should file a bug against CIL for this.

Reply:
I opened the issue https://github.com/goblint/cil/issues/40 and will remove the TODO from here then.